solidstart-tanstackquery-tailwind-modules/src/components/ProfileNav/ProfileNav.spec.tsx

```diff
@@ -0,0 +1,15 @@
+import { Router } from '@solidjs/router';
+import { render } from 'solid-testing-library';
+import { describe, expect, it } from 'vitest';
+import ProfileNav from './ProfileNav';
+
+describe('RepoCard for profilepage', () => {
+  it('should mount', async () => {
+    const wrapper = await render(() => (
+      <Router>
+        <ProfileNav class="border-none" pathname={'/'} />
+      </Router>
+    ));
+    expect(wrapper).toBeTruthy();
```

Comment:
I am not sure there is value in tests like this.

I don't think it tests anything, and I am not sure by the by the `describe` string what it is supposed to be testing

Reply:
Test will be updated and more test written